src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -208,20 +208,6 @@ export default {
 
 		// Get Lend Filter Exp version
 		this.getLendFilterExpVersion();
-
-		if (isFLSSEligible(this.$route)) {
```

Comment:
This extra check around the tracking call is the primary issue as the Algolia components immediately push in the `popularity` sort param causing the url read to be in-eligible by the time this is caused.